zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/signal/BroadcastSignalMultiplePartitionsTest.java

```diff
@@ -121,4 +139,117 @@ public void shouldTriggerMultipleSignalCatchEvent() {
             tuple("wf_2", ProcessInstanceIntent.ELEMENT_COMPLETING),
             tuple("wf_2", ProcessInstanceIntent.ELEMENT_COMPLETED));
   }
+
+  @Test
+  public void shouldAuthorizeOnAllPartitions() {
+    // given
+    final String signalName = newRandomSignal();
+    final String processId = Strings.newRandomValidBpmnId();
+    final String otherProcessId = Strings.newRandomValidBpmnId();
+    deployProcess(processId, "catch_main", signalName);
+    deployProcess(otherProcessId, "catch_other", signalName);
+
+    createProcessInstance(processId, 1);
+    createProcessInstance(otherProcessId, 2);
+
+    final UserRecordValue user = createUser();
+    grantProcessPermission(user.getUsername(), processId);
+
+    waitForSignalSubscriptions(signalName);
```

Comment:
🔧 I think this should wait for two different subscriptions: one for `processId` and one for `otherProcessId`.

Reply:
we are waiting for 2 subscriptions, the signal name is the same. We are waiting for 2 `SignalSubscriptionIntent.CREATED` one for each process